Marlin/ubl_motion.cpp

```diff
@@ -250,9 +231,8 @@
     const float m = dy / dx,
                 c = start[Y_AXIS] - m * start[X_AXIS];
 
-    const bool inf_normalized_flag = isinf(e_normalized_dist),
-               inf_m_flag = isinf(m);
-
+    const bool inf_normalized_flag=isinf(e_normalized_dist),
+               inf_m_flag=isinf(m);
```

Comment:
These are not pre-initialized variables, and neither are global `const` (or even `static const`) variables, necessarily, either.

In this case, code does have to execute, to assign the value to the variable at runtime. The only thing the `const` keyword does here is provides an optimization hint to the compiler, and inform the compiler not to allow the variable to be modified further on. It still behaves like any other stack-based variable at runtime.